zerver/actions/message_summary.py

```diff
@@ -120,7 +121,7 @@ def do_summarize_narrow(
         client_gravatar=True,
         allow_empty_topic_name=False,
         # Avoid fetching edit history, which won't be passed to the model.
-        allow_edit_history=False,
+        message_edit_history_visibility_policy=MessageEditHistoryVisibilityPolicyEnum.MESSAGE_EDIT_HISTORY_VISIBILITY_NONE,
```

Comment:
This is a bit absurdly long. I think the thing we can do is this -- no need for the values within the `Enum`, which are already namespaced, to repeat the type:

`message_edit_history_visibility_policy=MessageEditHistoryVisibilityPolicyEnum.VISIBILITY_NONE`